ts/parts-more/PackedBubbleSeries.ts

```diff
@@ -1113,14 +1113,13 @@ seriesType<Highcharts.PackedBubbleSeriesOptions>(
                 width: (series.parentNodeRadius as any) * 2,
                 height: (series.parentNodeRadius as any) * 2
             }, parentOptions);
-            if (!series.graph) {
+            if (!(series.parentNode as any).graphic) {
                 series.graph = (series.parentNode as any).graphic =
                     chart.renderer.symbol(parentOptions.symbol)
                         .attr(parentAttribs)
```

Comment:
Is that line necessary? It's done after if-else. Or some of `parentAttribs` are expected **before** adding to DOM?

Reply:
Thanks for the feedback! It was a relict after removing `if-else` from the code, I have already removed unnecessary `attr` in the newest commit.